backend/benefit/applications/tests/conftest.py

```diff
@@ -76,14 +77,21 @@ def talpa_service_with_one_application(talpa_service):
 
 @pytest.fixture
 def applications_csv_service():
+    # retrieve the objects through the default manager so that annotations are added
+    application1 = DecidedApplicationFactory(application_number=100001)
+    application2 = DecidedApplicationFactory(application_number=100002)
     return ApplicationsCsvService(
-        [DecidedApplicationFactory(), DecidedApplicationFactory()]
+        Application.objects.filter(pk__in=[application1.pk, application2.pk]).order_by(
+            "application_number"
+        )
     )
 
 
 @pytest.fixture
-def applications_csv_service_with_one_application():
-    return ApplicationsCsvService([DecidedApplicationFactory()])
+def applications_csv_service_with_one_application(applications_csv_service):
+    applications_csv_service.applications[1].delete()
+    applications_csv_service.applications = applications_csv_service.applications[:1]
+    return applications_csv_service
```

Comment:
What is the benefit here from using applications_csv_service as a base?
Would you feel this is better or worse:
```python
@pytest.fixture
def applications_csv_service_with_one_application():
    application1 = DecidedApplicationFactory(application_number=100001)
    return ApplicationsCsvService(Application.objects.filter(pk=application1.pk))

Reply:
hmm my idea was to do the creation of applications at one point... on the other hand, your suggestion makes applications_csv_service_with_one_application more self-contained. Changed